.circleci/build-requirements.txt

```diff
@@ -7,5 +7,4 @@ rsa==3.4.2
 paramiko==2.6.0
 boto3==1.10.0
 pyminizip===0.2.4
-git+https://github.com/demisto/demisto-py.git@v1.0.1 # disable-secrets-detection
```

Comment:
are you sure we don't need to add the new `demisto-py` here instead?

Reply:
we have it in `dev-requirements*.txt` file. I think it is ok.